src/main/cpp/blaze.cc

```diff
@@ -1236,8 +1236,15 @@ static void EnsureCorrectRunningVersion(BlazeServer *server) {
   string prev_installation;
   bool ok =
       blaze_util::ReadDirectorySymlink(installation_path, &prev_installation);
+  // The windows vesion of ReadDirectorySymlink will RealPath the
```

Comment:
This comment may get stale with time. Maybe start with "As of 2019-06-03, ..." or reference this PR.